code/model/modelread.cpp

```diff
@@ -1549,9 +1576,8 @@ int read_model_file(polymodel * pm, const char *filename, int n_subsystems, mode
 						turn_rate = static_cast<float>(atof(buf));
 					}
 
-					sm->dumb_turn_rate = turn_rate;
-				} else {
-					sm->dumb_turn_rate = 0.0f;
+					sm->default_turn_rate = turn_rate;
+					sm->flags.set(Model::Submodel_flags::Instant_rotate_accel);
```

Comment:
This should also set the ``Dumb_rotates`` flag, no?
Though it was missing before as well...

Reply:
Good call.

Reply:
Turns out it's not even needed (it's redundant in the one place it's used) so I removed it.